modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -15,13 +19,26 @@ public class PulsarContainer extends GenericContainer<PulsarContainer> {
 
     public static final String METRICS_ENDPOINT = "/metrics";
 
+    /**
+     * See <a href="https://github.com/apache/pulsar/blob/master/pulsar-common/src/main/java/org/apache/pulsar/common/naming/SystemTopicNames.java">SystemTopicNames</a>.
+     */
+    private static final String TRANSACTION_TOPIC_ENDPOINT =
+        "/admin/v2/persistent/pulsar/system/transaction_coordinator_assign/partitions";
+
     private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse("apachepulsar/pulsar");
 
     @Deprecated
-    private static final String DEFAULT_TAG = "2.2.0";
+    private static final String DEFAULT_TAG = "2.10.0";
+
+    /**
+     * Environment variable prefix needed to configure Pulsar standalone. See <a href="https://pulsar.apache.org/docs/next/getting-started-docker/">the guide</a>.
+     */
+    public static final String ENV_PULSAR_PREFIX = "PULSAR_PREFIX_";
```

Comment:
I think pulsar users will know about this and will be adding the prefix by themselves instead of relying on this constant.. IMHO, it should be removed.